modules/contracts-for-testing/src/main/kotlin/com/r3/corda/lib/tokens/testing/states/House.kt

```diff
@@ -15,5 +17,11 @@ data class House(
         val valuation: Amount<FiatCurrency>,
         override val maintainers: List<Party>,
         override val fractionDigits: Int = 5,
-        override val linearId: UniqueIdentifier = UniqueIdentifier()
-) : EvolvableTokenType()
\ No newline at end of file
+        override val linearId: UniqueIdentifier
+) : EvolvableTokenType()
+
+class Test : Contract {
```

Comment:
What is it?

Reply:
was a test to see if the absence of it was messing things upp... removed it.